iocBoot/iocIpmiMgr/st.cmd

```diff
@@ -16,8 +16,8 @@ drvAsynIPPortConfigure ("$(PORT)","$(IPADDR):623 udp",0,0,0)
 mchInit("$(PORT)")
 
 ## Load record instances
-dbLoadRecords("$(TOP)/db/asynRecord.db","P=${P}, R=${R}:asyn,PORT=$(PORT),ADDR=0,OMAX=80,IMAX=80")
-dbLoadRecords("${TOP}/db/microtca_bpm_crate.db", "link=$(PORT),P=$(P),R=$(R):,dev=$(P)$(R),unit=''")
+dbLoadRecords("$(TOP)/db/asynRecord.db","P=${AREA}:, R=asyn,PORT=$(PORT),ADDR=0,OMAX=80,IMAX=80")
```

Comment:
The asyn record here will have the wrong name, no? I think we need to define some value for `R` that's valid (I think using the MCH value for DIS would be valid)

Reply:
The name will be something like `IA-01RaBPM:asyn`. Do you think using the DIS value, for example, `IA-01RaBPM-CO:asyn`, would be a better approach?

Reply:
I was thinking of something closer to `IA-01RaBPM:CO-MCH_Base:asyn`

Reply:
But would it be hardcoded? Beacuse the substitution that adds the `MCH_Base` device is done in build time, but the substitution of `P` and `R` of asyn is done in runtime, when dbLoadRecords is called.